gatsby-config.js

```diff
@@ -81,9 +81,12 @@ module.exports = {
     `gatsby-plugin-smoothscroll`,
     `react-scrollspy`,
     {
-      resolve: `gatsby-plugin-google-analytics`,
+      resolve: `gatsby-plugin-gdpr-cookies`,
```

Comment:
Why the change?

Reply:
@juancarlosfarah  Basically, the new plugin only starts tracking using GA after the user presses "ok" in the cookie consent. This is for GDPR compliance.

Reply:
Great. You can go ahead and merge.